tensorflow_probability/python/mcmc/sample_test.py

```diff
@@ -56,6 +58,29 @@ def is_calibrated(self):
     return self._is_calibrated
 
 
+class RandomTransitionKernel(tfp.mcmc.TransitionKernel):
+  """Fake `TransitionKernel` that randomly assigns the next state.
+
+  Regardless of the current state, the `one_step` method will always
+  randomly sample from a Reyleigh Distribution.
+  """
+
+  def __init__(self, is_calibrated=True, accepts_seed=True):
+    self._is_calibrated = is_calibrated
+    self._accepts_seed = accepts_seed
```

Comment:
You don't actually use these args?

Reply:
Very true. The reason why I kept them was because the Transition Kernel needs to have an `is_calibrated` property anyways, and I wanted to set it up so that if this object is used for future tests, it's already all good to go. Alternatively, I could get rid of a formal `__init__`, remove the seed check in `one_step`, and have an `is_calibrated` property that does nothing (i.e. returns None). I think I prefer how it looks currently since it just seems more complete even if I might not be explicitly using those arguments. Thoughts?

Reply:
In general, it's probably best to only write code for something you have a clear plan for. Future proofing rarely works. That said, this is a minor thing, you can keep this as is if you prefer.

Reply:
Gotcha. Thanks for bringing it up though, I'll keep that in mind in the future!